game/Pl0000.h

```diff
@@ -92,58 +92,34 @@ class Pl0000 : public BehaviorAppBase
   float field_D34;
   int field_D38;
   int field_D3C;
-  int field_D40;
+  float field_D40;
   int field_D44;
-  int field_D48;
+  float field_D48;
   int field_D4C;
-  int field_D50;
-  int field_D54;
-  int field_D58;
+  float field_D50;
+  float field_D54;
+  float field_D58;
   int field_D5C;
-  int field_D60;
-  int field_D64;
-  int field_D68;
+  float field_D60;
+  float field_D64;
+  float field_D68;
   int field_D6C;
-  int field_D70;
-  int field_D74;
-  int field_D78;
-  int field_D7C;
+  float field_D70;
+  float field_D74;
+  float field_D78;
+  float field_D7C;
   D3DXMATRIX field_D80;
   D3DXMATRIX field_DC0;
   float field_E00;
   float field_E04;
-  int field_E08;
-  int field_E0C;
-  int field_E10;
-  int field_E14;
-  int field_E18;
-  int field_E1C;
-  int field_E20;
-  int field_E24;
-  int field_E28;
-  int field_E2C;
-  int field_E30;
-  int field_E34;
-  int field_E38;
-  int field_E3C;
-  int field_E40;
-  int field_E44;
-  int field_E48;
-  int field_E4C;
-  int field_E50;
-  int field_E54;
-  int field_E58;
-  int field_E5C;
-  int field_E60;
-  int field_E64;
-  int field_E68;
+  int inputBools[25];
   int field_E6C;
   int field_E70;
   int field_E74;
   int field_E78;
   int field_E7C;
-  float field_E80;
-  float field_E84;
+  float heavy_attack_count_up;
```

Comment:
same as comment in inputBools but should be like `m_fVariableName` and probably without _